### PR TITLE
upgraded gradle version

### PR DESCRIPTION
**Pull request by mbiarnes**
to have less warnings during a ./gradlew clean test

- compile had to be replaced by implementation
- the gradle version was upgraded to 7.0

**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[link](https://examle.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/gradlew [hunk 1]
```diff
@@ -28,16 +44,16 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (Ginxo) [inline job-dsls/gradlew:47]: just to assure this double quote thing is ok.... it looks weird and I'm wondering whether it should be `'-Xmx64m -Xms64m'` instead

**Review by Ginxo — APPROVED** [review]
just a small comment

File: job-dsls/gradlew (hunk 1 above)
Comment (mareknovotny) [reply job-dsls/gradlew:47]: isn't this change from updated gradlew script, i guess @mbiarnes didn't this manually

**Review by mareknovotny — APPROVED** [review]

File: job-dsls/gradlew (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/gradlew:47]: @Ginxo nop - this was automatic - looks strange - but it works

**Pull request merged**